sklearn/cluster/_bisect_k_means.py

```diff
@@ -337,7 +337,7 @@ def _bisect(self, X, x_squared_norms, sample_weight, cluster_to_bisect):
                 X, best_centers, best_labels, sample_weight
             )
         else:  # bisecting_strategy == "largest_cluster"
-            scores = np.bincount(best_labels)
+            scores = np.bincount(best_labels, minlength=2)
```

Comment:
Maybe add a comment above mentioning why we need minlength?

Reply:
Just to be sure: at this point we just have two possible labels because this is BisectingKMeans right?

Reply:
yes